tests/lib/nrf_modem_lib/lte_connectivity/src/lte_connectivity_test.c

```diff
@@ -24,6 +24,14 @@
 extern int unity_main(void);
 
 static pdn_event_handler_t pdn_event_handler_callback;
+static lte_lc_evt_handler_t lte_lc_event_handler_callback;
+
+/* Initial condition tracking */
+static bool initial_conditions_observed;
```

Comment:
This test is specifically to check what the flags are set to at boot time, and since these tests should be written so that they succeed no matter what order they are called in, there is no way to guarantee that the flags haven't been mutated by other tests except to cache them during setup